### PR TITLE
Improve code analysis

### PR DESCRIPTION
**Pull request by flenter**
This PR addresses two issues:

* add support for single quotes and template literals for hono routes
* added basic support for `@hono/zod-openapi`

### CODE REVIEW

**Review by mies — APPROVED** [review]

**Review by laulauland — APPROVED** [review]

File: packages/source-analysis/src/routeTrees/extractRouteTree.test.ts [hunk 1]
```diff
@@ -34,10 +39,17 @@ test.each([
...
   },
   {
     name: "import as",
     location: "../../test-cases/import-as",
+    expectedErrorCount: 0,
+  },
+  {
+    name: "zod-openapi",
+    location: "../../test-cases/zod-openapi",
+    expectedErrorCount: 2,
```
Comment (brettimus) [inline packages/source-analysis/src/routeTrees/extractRouteTree.test.ts:55]: unclear to me why the expectedErrorCount is 2 here! could you add a comment maybe?

**Review by brettimus — APPROVED** [review]
only had one comment on a new test case!

**Comment by brettimus** [comment]
oh also - changelog? though we haven't been updating it that faithfully after releases this week 😓

**Comment by flenter** [comment]
There's already an item in the (canary) changelog about improved code analysis. There was a typo in there though that i did fix

**Pull request merged**